docs/recitations/reci8-llms.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Recitation 8 - Large Language Models
+---
+# Recitation 8: Large Language Models
+
+## Setup Instructions (10 min)
+For this recitation and Project 4, you'll be using Google's Vertex AI Platform in a Jupyter notebook to query a large language model. First, we need to set up our GCP project to enable the API and install the prerequisites to run the notebook. 
+
+1.  Enable the Vertex AI API using [this link](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com). Make sure your NodeBB project from P3 is selected in the project selector dropdown.
+    
+2.  Install JupyterLab for Python following the instructions in [this link](https://jupyter.org/install).
+    
+3.  Install the gcloud CLI following the instructions in [this link](https://cloud.google.com/sdk/docs/install).
+    
+4.  In your terminal, run `gcloud auth login` and open the authentication URL that is displayed. Choose the account linked to the NodeBB project, and copy the verification code and paste it into the terminal.
```

Comment:
I had to run `gcloud auth application-default login` to fix auth problems in cell containing `query_llm`.

Note: I used homebrew to install the gcloud cli